evm.md

```diff
@@ -558,7 +558,7 @@ The `#next` operator executes a single step by:
  // ----------------------------------------------------
     rule <k> #badJumpDest? [ OP    ] => . ... </k> requires notBool isJumpOp(OP)
     rule <k> #badJumpDest? [ OP    ] => . ... </k> <wordStack> DEST  : WS </wordStack> <program> ... DEST |-> JUMPDEST ... </program> requires isJumpOp(OP)
-    rule <k> #badJumpDest? [ JUMPI ] => . ... </k> <wordStack> _ : 0 : WS </wordStack>
+    rule <k> #badJumpDest? [ JUMPI ] => . ... </k> <wordStack> _ : I : WS </wordStack> requires I ==Int 0
```

Comment:
hm, this looks strange .. .why is it needed?

Reply:
i's assuming in order to trigger the smt solver, but i think it should trigger it anyway, as part of the unification/matching decompositon

Reply:
yes, there is an issue of java backend regarding this, which i couldn't find a quick and/or nice fix.

Reply:
but i agree this should be fixed. i'll make a list of such issues to discuss later, either fixing the current java backend or using this info not to make the same mistake for the new backend.